documents/operating.md

```diff
@@ -11,6 +12,17 @@ Contents:
 6.  Policies
 
 ---------------------------------------------------------------------
+SECTION 0: 📓📓📓 Definitions 📓📓📓
+---------------------------------------------------------------------
+
+* **Bad Idea Factory Limited Liability Company ("the Company")** is a Limited Liability Company domiciled in Pennsylvania.
+* The Company may also be referred to as "**Bad Idea Factory**" or "**BIFFUD**".
+* "**Corporate Overlord**" shall mean an "owner" of the Company under Pennsylvania state law.
+* The "**Board of Directors**" ("the **Board**") of the Company, as defined by Pennsylvania state law, shall consist of all current Corporate Overlords of the Company.
+* "**Consensus**" shall mean TKTKTK.
+* "**Quorum**" shall mean TKTK
```

Comment:
Just a reminder -- % to pass has nothing to do with the number of people attending, it is a % of the COs in totality.

If 4 people aren't there, and those people don't vote in absentia, those 4 votes detract from the yes %.